cfme/tests/cloud/test_providers.py

```diff
@@ -959,4 +1038,34 @@ def test_create_azure_vm_from_azure_image():
                   VM is completely Stopped in Azure.
         title: Create Azure VM from Azure image
     """
-    pass
+    app = appliance.IPAppliance.from_url(vm_ip)
+
+    username = credentials['azure_appliance']['username']
```

Comment:
Is this a different credential than the one use for the powershell VM itself? I'm not quite clear what credential is being pulled here and used for the SSH client to that VM's IP.  Can this credential key be defined with the template name (see above comment about provider parametrization)

Reply:
@mshriver yes, it's different. I'm adding a comment there.